pkg/minikube/driver/driver.go

```diff
@@ -143,7 +143,12 @@ func BareMetal(name string) bool {
 
 // NeedsRoot returns true if driver needs to run with root privileges
 func NeedsRoot(name string) bool {
-	return name == None
```

Comment:
I think this can just be simplified to `return false` now.

none is only on Linux.

Reply:
done